patroni/postgresql.py

```diff
@@ -1367,16 +1369,16 @@ def save_configuration_files(self):
         try:
             for f in self._configuration_to_save:
                 if os.path.isfile(f):
-                    shutil.copy(f, f + '.backup')
+                    shutil.copy(f, f.replace(self._config_dir, self._data_dir) + '.backup')
```

Comment:
Actually one more thing, I think we need explicitly write locations of 'pg_hba.conf' and `pg_ident.conf` into `postgresql.conf`
https://github.com/jouir/patroni/blob/aaed71c51d46736aa0a6fc035e705ed73850e1ef/patroni/postgresql.py#L1057

```
hba_file = '/config/dir/pg_hba.conf'
ident_file = '/config/dir/pg_ident.conf'
```